### PR TITLE
Branch protection + workflow

### PR DESCRIPTION
**Pull request by kaczmarczyck**
My secrets and branch protection was not properly set up to push to main. Also, I use a marketplace solution for commit and push: `actions-js/push@master`

This PR has been tested with branch protection up on my fork, and uses the workaround to force push.

### CODE REVIEW

File: .github/workflows/create_summaries.yml [hunk 1]
```diff
@@ -37,18 +39,14 @@ jobs:
...
-          git add fuzzing_results/summaries/
-          git commit -a -m "update summaries"
-          git push origin main
-        else
-          echo 'There were no new reports.'
-        fi
+        git add results/summaries/
+        git add fuzzing_results/summaries/
+    - name: Commit & Push changes
+      uses: actions-js/push@master
```
Comment (jmichelp) [inline .github/workflows/create_summaries.yml:45]: It's forbidden by company policy to point github actions to a branch HEAD.
Please use either a tag (preferred) or a commit id instead

File: .github/workflows/create_summaries.yml [hunk 2]
```diff
@@ -37,18 +39,14 @@ jobs:
...
-        git config user.name "kaczmarczyck"
-        if [[ `git status --porcelain` ]]; then
-          git add results/summaries/
-          git add fuzzing_results/summaries/
-          git commit -a -m "update summaries"
-          git push origin main
-        else
-          echo 'There were no new reports.'
-        fi
+        git add results/summaries/
```
Comment (jmichelp) [inline .github/workflows/create_summaries.yml:42]: nit: I'm pretty sure you can put both path on a single `git add` command. Your call

Comment (kaczmarczyck) [reply .github/workflows/create_summaries.yml]: Sure, why not, changed.

File: .github/workflows/create_summaries.yml (hunk 1 above)
Comment (kaczmarczyck) [reply .github/workflows/create_summaries.yml]: Makes sense, good to know!

**Review by jmichelp — APPROVED** [review]

**Pull request merged**